languages/python/polar/polar.py

```diff
@@ -22,13 +22,17 @@ class Polar:
 
     def __init__(self, classes=CLASSES, constructors=CONSTRUCTORS):
         self.ffi_polar = lib.polar_new()
-        self.host = Host(self.ffi_polar, classes=classes, constructors=constructors)
+        self.host = Host(self.ffi_polar)
         self.load_queue = []
 
         # Register built-in classes.
         self.register_class(datetime, name="Datetime")
         self.register_class(timedelta, name="Timedelta")
 
+        # Pre-registered classes.
+        for name, cls in classes.items():
+            self.register_class(cls, name=name, from_polar=constructors[name])
```

Comment:
Should the equivalent change be made for Ruby and Java?

Reply:
No, because they don't have the decorator that frobs the globals that this handles. They always explicitly use `register_class` after the `Polar` instance is instantiated.